mlflow/ml-package-versions.yml

```diff
@@ -2,8 +2,10 @@ sklearn:
   package_info:
     pip_release: "scikit-learn"
     install_dev: |
-      # This installs scikit-learn from the default branch
-      pip install git+https://github.com/scikit-learn/scikit-learn.git
+      # TODO: Remove `pip install Cython numpy scipy` and `--no-use-pep517` once
+      #       https://github.com/scikit-learn/scikit-learn/issues/20189 is fixed
+      pip install Cython numpy scipy
+      pip install --no-use-pep517 git+https://github.com/scikit-learn/scikit-learn.git
```

Comment:
As described in https://github.com/scikit-learn/scikit-learn/issues/20189, installing with PEP 517 seems to be the root cause. As a workaround, use `--no-use-pep517` option.